### PR TITLE
Add a remix example

### PR DESCRIPTION
**Pull request by glommer**
This is the blog posts example from Remix.
This simple version only uses CRUD endpoints. The goal in the near future is to add custom endpoints, 
authentication and seeding.

### CODE REVIEW

**Comment by penberg** [comment]
@glommer Did you keep the Prisma bits there on purpose? I would drop them because it can make it harder for folks to digest the example with both Prisma and ChiselStrike code in there. Also, the `.github` files should probably just go.

**Review by dekimir — APPROVED** [review]
Just two nitpicks.

**Comment by glommer** [comment]
> @glommer Did you keep the Prisma bits there on purpose? I would drop them because it can make it harder for folks to digest the example with both Prisma and ChiselStrike code in there. Also, the `.github` files should probably just go.

I did, and mentioned in the readme. The Remix blog example in their website has users and other things. But the example in the website takes you through adding blog posts. I, on purpose, implemented just what's in their website.

In the future we can replace more things.

File: remix/app/models/post.server.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	    return crud.results;
+	});
+  });
+}
+
+// The second function will just call the POST endpoint and create an object.
+export async function createPost(post) {
+  const url = chiselUrl("posts")
+  return fetch(url, {
+         method: 'post',
```
[the remark this answers is not in the document]
Comment (glommer) [reply remix/app/models/post.server.ts:44]: I'll run a linter.

[the remark this answers is not in the document]
Comment (glommer) [reply remix/app/models/post.server.ts:44]: I'll be uploading the result of running `npm run format`,

File: remix/app/models/post.server.ts [hunk 2]
```diff
@@ -0,0 +1,58 @@
+// We import the ChiselStrike type directly and can reuse it in here. 
+// It should also be possible to do the other way around: define the type
+// in the frontend, and get ChiselStrike to consume it.
+//
+// That is https://github.com/chiselstrike/chiselstrike/issues/1589. Could
+// that be your first issue??
```
[the remark this answers is not in the document]
Comment (glommer) [reply remix/app/models/post.server.ts:6]: Changed the text.
Ident is now whatever `npm run format` changed it to.

**Review by dekimir — APPROVED** [review]

**Pull request merged**